src/Session.php

```diff
@@ -55,10 +55,12 @@ public function isStarted()
     }
 
     /**
-     * Starts session driver.
+     * Starts session driver if session is not started.
      *
      * Calling any action before visiting a page is an undefined behavior.
      * The only supported method calls on a fresh driver are
+     * - resizeWindow()
+     * - maximizeWindow()
```

Comment:
Oh okay. So what do you think about: https://github.com/minkphp/driver-testsuite/pull/10